lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -3189,6 +3189,8 @@ SPIRVValue *LLVMToSPIRVBase::transIntrinsicInst(IntrinsicInst *II,
   case Intrinsic::invariant_end:
   case Intrinsic::dbg_label:
   case Intrinsic::trap:
+  // llvm.instrprof.increment is not supported
+  case Intrinsic::instrprof_increment:
```

Comment:
There are more `llvm.instrprof.*` intrinsics. I believe, if we want to add this one we should add the others too.